### PR TITLE
PHP8.4 announcement in french

### PR DESCRIPTION
**Pull request by adoy**

### CODE REVIEW

File: releases/8.4/common.php [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 const LANGUAGES = [
     'en' => 'English',
+    'fr' => 'Français',
```
Comment (adoy) [inline releases/8.4/common.php:11]: Not sure what should be the actual real order there. Should we have some kind of "rule" ?

File: releases/8.4/languages/fr.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    'dom_additions_html5_title' => 'Nouvelles fonctionnalités de l\'extension ext-dom et prise en charge de HTML5.',
+    'dom_additions_html5_description' => '<p>Nouvelle API DOM offrant une prise en charge conforme aux standards pour l\'analyse des documents HTML5, corrigeant plusieurs bogues de conformité de longue date dans le comportement des fonctionnalités DOM et ajoutant plusieurs fonctions pour faciliter la manipulation des documents.</p><p>La nouvelle API DOM est disponible dans l\'espace de noms <code>Dom</code>. Les documents utilisant cette API peuvent être créés à l\'aide des classes <code>Dom\HTMLDocument</code> et <code>Dom\XMLDocument</code>.</p>',
+    'new_array_find_title' => 'Nouvelles fonctions <code>array_*()</code>',
+    'new_array_find_description' => 'Les nouvelles fonctions <a href="/manual/en/function.array-find.php"><code>array_find()</code></a>, <a href="/manual/en/function.array-find-key.php"><code>array_find_key()</code></a>, <a href="/manual/en/function.array-any.php"><code>array_any()</code></a> et <a href="/manual/en/function.array-all.php"><code>array_all()</code></a> sont désormais disponibles.',
```
Comment (saundefined) [inline releases/8.4/languages/fr.php:19]:
```suggestion
    'new_array_find_description' => 'Les nouvelles fonctions <a href="/manual/fr/function.array-find.php"><code>array_find()</code></a>, <a href="/manual/fr/function.array-find-key.php"><code>array_find_key()</code></a>, <a href="/manual/fr/function.array-any.php"><code>array_any()</code></a> et <a href="/manual/fr/function.array-all.php"><code>array_all()</code></a> sont désormais disponibles.',
```

File: releases/8.4/languages/fr.php [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    'bc_typed_constants' => 'Les constantes de classe des extensions <code>date</code>, <code>intl</code>, <code>pdo</code>, <code>reflection</code>, <code>spl</code>, <code>sqlite</code>, <code>xmlreader</code> sont désormais typées.',
+    'bc_mysqli_constants' => 'Les constantes <code>MYSQLI_SET_CHARSET_DIR</code>, <code>MYSQLI_STMT_ATTR_PREFETCH_ROWS</code>, <code>MYSQLI_CURSOR_TYPE_FOR_UPDATE</code>, <code>MYSQLI_CURSOR_TYPE_SCROLLABLE</code>, et <code>MYSQLI_TYPE_INTERVAL</code> ont été supprimées.',
+    'bc_mysqli_functions' => 'Les fonctions <code>mysqli_ping()</code>, <code>mysqli_kill()</code>, <code>mysqli_refresh()</code>, méthodes <code>mysqli::ping()</code>, <code>mysqli::kill()</code>, <code>mysqli::refresh()</code>, et constantes <code>MYSQLI_REFRESH_*</code> sont désormais obsolètes.',
+    'bc_standard' => '<code>stream_bucket_make_writeable()</code> et <code>stream_bucket_new()</code> renvoient désormais une instance de <code>StreamBucket</code> au lieu de <code>stdClass</code>.',
+    'bc_core' => '<a href="/manual/en/migration84.incompatible.php#migration84.incompatible.core.exit">Changement de comportement de la fonction <code>exit()</code></a>.',
```
Comment (saundefined) [inline releases/8.4/languages/fr.php:49]:
```suggestion
    'bc_core' => '<a href="/manual/fr/migration84.incompatible.php#migration84.incompatible.core.exit">Changement de comportement de la fonction <code>exit()</code></a>.',
```

File: releases/8.4/languages/fr.php [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    'bc_warnings' => 'La constante <code>E_STRICT</code> est désormais obsolète.',
+
+    'footer_title' => 'Meilleures performances, meilleure syntaxe, sécurité des types améliorée.',
+    'footer_description' => '<p>Pour télécharger les sources de PHP 8.4, veuillez visiter la page des <a href="/downloads">téléchargements</a>. Les binaires pour Windows sont disponibles sur le site <a href="https://windows.php.net/download">PHP for Windows</a>. La liste des changements est enregistrée dans le <a href="/ChangeLog-8.php#PHP_8_4">ChangeLog</a>.</p> <p>Le <a href="/manual/en/migration84.php">guide de migration</a> est disponible dans le manuel PHP. Veuillez le consulter pour une liste détaillée des nouvelles fonctionnalités et des changements non compatibles avec les versions précédentes.</p>',
```
Comment (saundefined) [inline releases/8.4/languages/fr.php:53]:
```suggestion
    'footer_description' => '<p>Pour télécharger les sources de PHP 8.4, veuillez visiter la page des <a href="/downloads">téléchargements</a>. Les binaires pour Windows sont disponibles sur le site <a href="https://windows.php.net/download">PHP for Windows</a>. La liste des changements est enregistrée dans le <a href="/ChangeLog-8.php#PHP_8_4">ChangeLog</a>.</p> <p>Le <a href="/manual/fr/migration84.php">guide de migration</a> est disponible dans le manuel PHP. Veuillez le consulter pour une liste détaillée des nouvelles fonctionnalités et des changements non compatibles avec les versions précédentes.</p>',
```

**Review by saundefined — APPROVED** [review]

**Pull request merged**